doc/articles/get-started-vs-2022.md

```diff
@@ -50,7 +50,8 @@ To easily create a multi-platform application:
 * Create a new C# solution using the **Multi-Platform App (Uno Platform|Xamarin)** template, from Visual Studio's **Start Page**:
 
 ![new project](Assets/quick-start/vsix-new-project.png)
-* Update to the latest NuGet package named `Uno.UI`. To get the very latest preview features, check the `pre-release` box.
+* Right click on the Solution and select `Manage NuGet Packages for Solution` from the context menu.
+* Update to the latest NuGet package named `Uno.UI`. To get the very latest preview features, check the `Include prerelease` box.
```

Comment:
Avoid having "prerelease"/"preview" topics in the getting started guide. I'd suggest we have an additional page that covers "latest and greatest".
We need to move away from "experimental framework" to "production/stable platform"